solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -343,17 +345,45 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
     }
 
     /**
-     * @dev returns the reserve balances of the given reserve tokens
+     * @dev transfers a portion of the accumulated conversion fees
+     */
+    function processNetworkFees() external protected {
+        (uint256 reserveBalance0, uint256 reserveBalance1) = processNetworkFees(0);
+        _reserveBalancesProduct = reserveBalance0 * reserveBalance1;
+    }
+
+    /**
+     * @dev transfers a portion of the accumulated conversion fees
+     *
+     * @param _value amount of ether to exclude from the ether reserve balance (if relevant)
+     *
+     * @return new reserve balances
+     */
+    function processNetworkFees(uint256 _value) internal returns (uint256, uint256) {
+        syncReserveBalances(_value);
+        (uint256 reserveBalance0, uint256 reserveBalance1) = reserveBalances(1, 2);
+        (address wallet, uint256 fee0, uint256 fee1) = networkWalletAndFees(reserveBalance0, reserveBalance1);
+        reserveBalance0 -= fee0;
```

Comment:
I'd add underflow protection here just in case

Reply:
Function `networkWalletAndFees` takes care of that.
If you find this insufficient, then we have the same problem in function `baseReserveBalances` (which also calls function `networkWalletAndFees`).